client/state/reader-ui/sidebar/reducer.js

```diff
@@ -44,9 +45,19 @@ export const openOrganizations = withPersistence( ( state = [], action ) => {
 	return state;
 } );
 
+export const isRecentOpen = withPersistence( ( state = false, action ) => {
+	switch ( action.type ) {
+		case READER_SIDEBAR_RECENT_TOGGLE:
+			return ! state;
+		default:
+			return state;
```

Comment:
Similar convention question here. I prefer this style but the other reducers are not providing a `default` branch of the `switch` and instead are just returning the fallback value outside of the `switch`. Not a big deal but it might be worth keeping this consistent with them or updating all of them to match this style.